content/blog/2020-08-10-darktable-3.2/2020-08-10-darktable-3.2.md

```diff
@@ -0,0 +1,462 @@
+
+It has been noticed that bugs, sometimes important, are present in version 3.0.x (up to 3.0.2). Especially bugs related to the display of images, especially in case of multiple instances (but not only). Most of these bugs resulted from the new image processing order (pipeline), integrated in version 3.0. A complete overhaul of the processing pipeline has been done by Pascal Obry for a more robust management of the new pipeline integrated in 3.0. This overhaul makes the pipeline both more stable and more efficient.
+
+Image modification detection has been improved. This mainly avoids the need to recalculate the display of an image that has not been altered, for example by having been opened in a darkroom without any modification. The modification of XMPs and thumbnails is thus more reliable and only impacted in the event of a real modification.
+
+Many other performance improvements have been integrated in this version 3.2, especially in some modules. This work is in progress since several other proposals for performance enhancements in other modules are already in progress for integration in the future 3.4. Thus, the next 3.4 seems to be shaping up with significant performance improvements.
+
+darktable 3.2 also introduces an automatic database maintenance option (configurable in the preferences), allowing a healthier database to be maintained over time.
+
+Finally, with the various code improvements, many similar parts have been unified, allowing later a faster and easier integration of new modules and a much better consistency on similar parts (as, for example, what has been done on pickers, described above).
+Test tools have also been added, making it easier for developers to test new additions and the backward compatibility and stability of your image processing, especially when rewriting a module for better performance. 
+
+And of course, many bugs have been fixed, making this 3.2 the most stable version 3 to date. And very clearly. If you are already using a 3.0.x version, I can just recommend you to upgrade to this 3.2, if only for its superior stability.
+
+## Some new options in preferences window
+
+Apart from the new workflows presented above and the few preferences already presented, you will find some new preferences options that may be very useful. Here are the main ones:
+
+- On lighttable, you can sort film roll by number or by folder. Following the redesign of lighttable, you can also define different size categories (via size category separators) which will allow you to define a different overlay mode according to the size of the thumbnails. The size is specified in pixels and the thumbnail size is displayed in the overlay mode popover menu shown above.
+
+- In _darkroom_, it is now possible to reduce the resolution of the navigation preview, reducing CPU usage and possibly improving performance.
+**_Warning :_** as describes in the tooltip of this option, this can reduce slightly guided mask accuracy.
```

Comment:
"... as **described** in the tooltip [...] this can **slightly reduce** the guided mask accuracy."